Detectors/MUON/MCH/Raw/ImplHelpers/DumpBuffer.h

```diff
@@ -106,10 +105,12 @@ void dumpWordInfo<o2::mch::raw::UserLogicFormat>(std::ostream& out, uint64_t w,
     if (!o2::mch::raw::isSampaSync(w)) {
       out << spacer;
     }
+    ULHeaderWord<VERSION> header{w};
+    int gbt = header.linkID;
+    int elinkid = header.dsID;
+    int error = header.error | (header.incomplete << 2);
```

Comment:
Again, here we are forcing the incomplete bit to be bit 2, while in V1 format it is actually bit 3. I don't know however if this is a problem in this context.